docs/advanced.rst

```diff
@@ -1562,3 +1562,27 @@ work, it is important that all lines are indented consistently, i.e.:
 
 .. [#f4] http://www.sphinx-doc.org
 .. [#f5] http://github.com/pybind/python_example
+
+Calling Python from C++
+=======================
+
+Pybind11 also allows to call python code from C++.
+
+.. code-block:: cpp
+
+    // initialize the interpreter.
+	Py_Initialize() ;
```

Comment:
You won't need it if example18 is part of the extension module, as are example1-17.